sources/packages/backend/apps/queue-consumers/src/services/clamav/clamav.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable, Logger } from "@nestjs/common";
+import * as NodeClam from "clamscan";
+import { Readable } from "stream";
+
+@Injectable()
+export class ClamAVService {
+  private logger = new Logger(ClamAVService.name);
+  private scanner: NodeClam;
+
+  /**
+   * Initialize the scanner.
+   */
+  private async initClam() {
+    this.scanner = await new NodeClam().init({
+      clamdscan: {
+        host: process.env.CLAMAV_HOST,
+        port: Number(process.env.CLAMAV_PORT),
```

Comment:
is this required to be explicitly changed to number before assigning?

Reply:
Yes